src/wp-includes/theme.php

```diff
@@ -840,6 +841,26 @@ function switch_theme( $stylesheet ) {
 	$new_theme->delete_pattern_cache();
 	$old_theme->delete_pattern_cache();
 
+	// Set autoload=no for the previous all themes.
+	$theme_mods_compare_value = '%' . $wpdb->esc_like( 'theme_mods_' ) . '%';
```

Comment:
No need or the wildcard at the beginning, right? The options are always prefixed with `theme_mod_`, right?

```suggestion
	$theme_mods_compare_value = $wpdb->esc_like( 'theme_mods_' ) . '%';
```